sql/size_utils.sql

```diff
@@ -682,3 +635,125 @@ BEGIN
 END;
 $BODY$;
 
+-------------Get index size for hypertables -------
+--schema_name      - schema_name for hypertable index
+-- index_name      - index on hyper table
+---note that the query matches against the hypertable's schema name as
+-- the input is on the hypertable index nd not the chunk index.
+CREATE OR REPLACE FUNCTION _timescaledb_internal.indexes_local_size(
+    schema_name_in             NAME,
+    index_name_in              NAME
+)
+RETURNS TABLE ( hypertable_id INTEGER,
+                total_bytes BIGINT ) 
+               LANGUAGE PLPGSQL STABLE STRICT
+               AS
+$BODY$
+BEGIN
+        RETURN QUERY
+        SELECT ci.hypertable_id, sum(pg_relation_size(c.oid))::bigint
```

Comment:
Since this is just a select, does it have to be `plpgsql`?

Reply:
No, it doesn't.  But don't see any special benefit from using plain SQL either.